packages/embark/src/lib/modules/whisper/index.js

```diff
@@ -18,6 +18,7 @@ class Whisper {
     this.web3 = new Web3();
     this.embark = embark;
     this.web3Ready = false;
+    this.webSocketsChannels = [];
```

Comment:
```suggestion
    this.webSocketsChannels = {};
```
Should it be an object in view of [L216](https://github.com/embark-framework/embark/blob/2cbfaf59c315e912a809e70da0339409fb6b5965/packages/embark/src/lib/modules/whisper/index.js#L216) and [L214](https://github.com/embark-framework/embark/blob/2cbfaf59c315e912a809e70da0339409fb6b5965/packages/embark/src/lib/modules/whisper/index.js#L214)?

Reply:
Changed the array to an object. Good catch @michaelsbradleyjr!